framework_lib/src/commandline/mod.rs

```diff
@@ -746,6 +748,21 @@ pub fn run_with_args(args: &Cli, _allupdate: bool) -> i32 {
     } else if let Some(Some(kblight)) = args.kblight {
         assert!(kblight <= 100);
         ec.set_keyboard_backlight(kblight);
+    } else if !args.rgbkbd.is_empty() {
+        if args.rgbkbd.len() < 2 {
+            println!(
+                "Must provide at least 2 arguments. Provided only: {}",
+                args.rgbkbd.len()
+            );
+        } else {
+            let start_key = args.rgbkbd[0] as u8;
+            let colors = args.rgbkbd[1..].iter().map(|color| RgbS {
```

Comment:
bonus points if you figure out how to do the *8 multipliers like i have on the ec console.

Reply:
left as an exercise for the reader